SpectraLogic.EscapePodClient/Exceptions/UnreachableEndpointException.cs

```diff
@@ -0,0 +1,34 @@
+﻿/*
+ * ******************************************************************************
+ *   Copyright 2014-2018 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+using System;
+
+namespace SpectraLogic.EscapePodClient.Exceptions
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <seealso cref="System.Exception" />
+    public class UnreachableEndpointException : Exception
```

Comment:
When is this exception thrown?

Reply:
this is just a placeholder for any network exception, I didn't know how you are going to handle this, so I created a stub endpoint exception